playground/main.ts

```diff
@@ -1,5 +1,13 @@
 import { storyblokInit, loadStoryblokBridge } from "@storyblok/js";
 
+declare global {
+  interface Window {
+    initWithBridge: any;
```

Comment:
Can we add strict typing here? Just asking, maybe `any` is too generic, but it can be necessary. Your call. Same question for the following props.